Tools/autotest/sim_vehicle.py

```diff
@@ -51,71 +55,78 @@ def parse_args(self):
 
             args = []
             opts = e.opts
-            mavproxy_args = []
-            mavproxy_args.append(str(e)[16:]) # this trims "no such option" off
+            mavproxy_args = [str(e)[16:]]  # this trims "no such option" off
             mavproxy_args.extend(e.rargs)
             opts.ensure_value("mavproxy_args", " ".join(mavproxy_args))
 
         return opts, args
 
-def cygwin_pidof(procName):
-    ''' Thanks to kata198 for this:
+
+def cygwin_pidof(proc_name):
+    """ Thanks to kata198 for this:
     https://github.com/kata198/cygwin-ps-misc/blob/master/pidof
-    '''
-    pipe = subprocess.Popen("ps -ea | grep " + procName, shell=True, stdout=subprocess.PIPE)
-    outputLines = pipe.stdout.read().replace("\r", "").split("\n")
+    """
+    pipe = subprocess.Popen("ps -ea | grep " + proc_name, shell=True, stdout=subprocess.PIPE)
+    output_lines = pipe.stdout.read().replace("\r", "").split("\n")
     ret = pipe.wait()
     pids = []
     if ret != 0:
         # No results
-	return []
-    for line in outputLines:
-	if not line:
-	    continue
-	lineSplit = [item for item in line.split(' ') if item]
-	cmd = lineSplit[-1].split('/')[-1]
-	if cmd == procName:
-	    try:
-		pid = int(lineSplit[0].strip())
-	    except:
-		pid = int(lineSplit[1].strip())
-		pid = str(pid)
-	    if pid not in pids:
-		pids.append(pid)
+        return []
+    for line in output_lines:
+        if not line:
+            continue
+        line_split = [item for item in line.split(' ') if item]
+        cmd = line_split[-1].split('/')[-1]
+        if cmd == proc_name:
+            try:
+                pid = int(line_split[0].strip())
+            except:
+                pid = int(line_split[1].strip())
+                str_pid = str(pid)
+            if str_pid not in pids:
+                pids.append(str_pid)
     return pids
 
+
 def under_cygwin():
+    """Return Cygwin path"""
```

Comment:
This doesn't return the Cygwin path, it returns the existence of the cygwin binary